Jolt/Math/UVec4.inl

```diff
@@ -105,23 +105,31 @@ UVec4 UVec4::sGatherInt4(const uint32 *inBase, UVec4Arg inOffsets)
 
 UVec4 UVec4::sMin(UVec4Arg inV1, UVec4Arg inV2)
 {
-#if defined(JPH_USE_SSE)
+#if defined(JPH_USE_SSE4_1)
 	return _mm_min_epu32(inV1.mValue, inV2.mValue);
 #elif defined(JPH_USE_NEON)
 	return vminq_u32(inV1.mValue, inV2.mValue);
 #else
-	#error Unsupported CPU architecture
+	UVec4 result;
+	for (int i = 0; i < 4; i++) {
```

Comment:
Jolt uses either:

    for (...)
    {
         multiple;
         statements;
    }

or

    for (....)
      single_statement;

Could you please remove the brackets? (here and also for the other functions you changed)